x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/service_overview/instances_detailed_statistics.spec.ts

```diff
@@ -49,8 +61,166 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
         });
       });
 
-      // UNSUPPORTED TEST CASES - when data is loaded
-      // TODO: These tests should be migrated to use synthtrace: https://github.com/elastic/kibana/issues/200743
+      describe('when data is loaded', () => {
+        let apmSynthtraceEsClient: ApmSynthtraceEsClient;
+
+        before(async () => {
+          apmSynthtraceEsClient = await synthtrace.createApmSynthtraceEsClient();
+          await generateServiceData({
+            apmSynthtraceEsClient,
+            start: new Date(start).getTime(),
+            end: new Date(end).getTime(),
+            name: serviceName,
+            environment: 'ENVIRONMENT_ALL',
+            agentName: 'java',
+          });
+        });
+
+        after(() => apmSynthtraceEsClient.clean());
+
+        describe('fetching data without comparison', () => {
+          let response: Response;
+          let serviceNodeIds: string[];
+
+          beforeEach(async () => {
+            serviceNodeIds = await getServiceNodeIds({
+              apmApiClient,
+              start,
+              end,
+            });
+
+            response = await apmApiClient.readUser({
+              endpoint:
+                'GET /internal/apm/services/{serviceName}/service_overview_instances/detailed_statistics',
+              params: {
+                path: { serviceName },
+                query: {
+                  latencyAggregationType: LatencyAggregationType.avg,
+                  start,
+                  end,
+                  numBuckets: 20,
+                  transactionType: 'request',
+                  serviceNodeIds: JSON.stringify(serviceNodeIds),
+                  environment: 'ENVIRONMENT_ALL',
+                  kuery: '',
+                },
+              },
+            });
+          });
+
+          it('returns a service node item', () => {
+            expect(Object.values(response.body.currentPeriod).length).to.be.greaterThan(0);
+            expect(Object.values(response.body.previousPeriod)).to.eql(0);
+          });
+
+          it('returns statistics for each service node', async () => {
```

Comment:
Q Do you need it to be `async` here? Maybe we can remove it, wdyt?
```suggestion
          it('returns statistics for each service node', () => {
```